packages/components/molecules/f-rating/src/components/Rating.vue

```diff
@@ -111,44 +133,67 @@ export default {
         getRatingStarPercentage () {
             return `${(this.starRating / this.maxStarRating) * 100}%`;
         }
+    },
+
+    methods: {
+        /**
+         * Display review rating message format.
+         *
+         * @todo - If the component is using `short` as a `starRatingSize` we shouldn't display text
+         * alongside it for now. (TBC with design - ticket in backlog).
+         *
+         * @returns {string|boolean}
+         */
+        getRatingDisplayFormat () {
+            return this.ratingDisplayType
+                ?
+                this.$t(`ratings.ratingDisplayType.${this.ratingDisplayType}`, {
+                    rating: this.starRating,
+                    maxStarRating: this.maxStarRating,
+                    totalReviews: this.totalReviews
+                })
+                : false;
```

Comment:
I'd caution against returning a boolean here if the other return type is a string. Can lead to a bit of confusion down the line! I'd just return the copy and rely on other logic to decide if it should be used or not

Reply:
Removed.